src/v0/destinations/pinterest_tag/transform.js

```diff
@@ -119,25 +137,44 @@ const process = (event) => {
   }
 
   toSendEvents.forEach((sendEvent) => {
-    respList.push(responseBuilderSimple(sendEvent));
+    respList.push(responseBuilderSimple(sendEvent, destination));
   });
   return respList;
 };
 
-const generateBatchedPayloadForArray = (events) => {
+const generateBatchedPayloadForArray = (events, { Config }) => {
+  const { apiVersion, adAccountId, conversionToken } = Config;
+  const endpoint = getV5EventsEndpoint(adAccountId);
```

Comment:
can we confirm that even if v3 is chosen, adAccountId will be provided by the user? if we cannot then let's move this line after line no 155